libcodechecker/server/config_database.py

```diff
@@ -4,41 +4,37 @@
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
 """
-Database server handling.
+Database server handling for the server's configuration database.
```

Comment:
Because it uses entirely different semantics in subtle places. First of all, the used schema is imported from a different place. Second of all, failure to instantiate the config connection is a critical condition that brings down the entire server -- failure to instantiate a product connection is only an error. Config connection is initiated from the command-line, whereas product connections are created from connection strings.

From above, the two files look very similar. But there are many subtle differences. Just as how we have duplicated the common parts in the schema (such as the Metatable), and duplicate the migration environment's common code, I think it's much better if the two database connection systems are separated on the package name.